source/command-monitoring/tests/unified/command.yml

```diff
@@ -0,0 +1,94 @@
+description: "commands"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
+  - database:
+      id: &database database
+      client: *client
+      databaseName: &databaseName command-monitoring-tests
+  - collection:
+      id: &collection collection
+      database: *database
+      collectionName: &collectionName test
+
+initialData:
+  - collectionName: *collectionName
+    databaseName: *databaseName
+    documents:
+      - { _id: 1, x: 11 }
+
+tests:
+  - description: "A successful command"
+    operations:
+      - name: distinct
+        object: *collection
+        arguments:
+          fieldName: x
+          filter: {}
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                distinct: *collectionName
+                key: x
+              commandName: distinct
+              databaseName: *databaseName
+          - commandSucceededEvent:
+              reply:
+                ok: 1
+                values:
+                  - 11
+              commandName: distinct
+
+  - description: "A failed command"
+    operations:
+      - name: find
+        object: *collection
+        arguments:
+          filter: { $or: true }
+        expectError:
+          isClientError: false
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                find: *collectionName
+                filter: { $or: true }
+              commandName: find
+              databaseName: *databaseName
+          - commandFailedEvent:
+              commandName: find
+
+  - description: "A successful command with a non-primary read preference"
+    operations:
+      - name: distinct
+        object: *collection
+        arguments:
+          fieldName: x
+          filter: {}
+          readPreference:
+            mode: primaryPreferred
```

Comment:
I believe this may be one of the first unified spec tests that specifies a `readPreference` argument on an operation. This is permitted per [Special Handling for Arguments](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#special-handling-for-arguments) but I wanted to confirm that it makes sense to do so.

The APM assertion below doesn't actually care about this option being specified, so perhaps we can remove it.

Reply:
I think this test is meant to check drivers correctly handle this spec requirement: https://github.com/mongodb/specifications/blob/master/source/command-monitoring/command-monitoring.rst#read-preference

that said, I think that is only relevant when drivers are using OP_QUERY and the $query wrapping of command arguments applies. so maybe it is not worth keeping around?

Reply:
I'd rather delete this than keep a pre-3.6 test around. Especially if the Node team doesn't doesn't care.

Reply:
any opinion on keeping or deleting this test @durran?

Reply:
While the Node 3.7 still supports pre 3.6 servers and this works, being that our current release is 4.6.0 we wouldn't backport these changes. I'd be in preference of deleting this test.

Reply:
Java and Go still support pre-3.6 servers, but I still think we should make little if any effort to maintain pre-3.6 tests